pkg/backend/local/display.go

```diff
@@ -313,6 +314,11 @@ func shouldShow(step engine.StepEventMetadata, opts backend.DisplayOptions) bool
 		return opts.ShowSameResources
 	}
 
+	// If this step refers to a default provider resource, we do not display any changes.
+	if providers.IsProviderType(step.URN.Type()) && step.URN.Name() == "default" {
+		return false
+	}
```

Comment:
should we consider putting this as the first thing we check in thsi function?  right now, if we have a default provider, and an opsame and showsameresources=true, then we'll show things.  Is that what we want?  Or should we always hide a default provider?

Reply:
Good catch. These should always be hidden.